.gitattributes

```diff
@@ -3,4 +3,5 @@
 
 compiler-tests/**/*.java linguist-generated=true
 **/yarn.lock linguist-generated=true
-**/package-lock.lock linguist-generated=true
\ No newline at end of file
+**/package-lock.lock linguist-generated=true
+*.bat text eol=crlf
```

Comment:
Happy to merge but as a non-windows user can you tell me what this is for for my own awareness 😅